components/brave_wallet_ui/common/hooks/swap.ts

```diff
@@ -78,6 +78,7 @@ const hasDecimalsOverflow = (amount: string, asset?: BraveWallet.BlockchainToken
   }
 
   const amountBaseWrapped = new Amount(amount).multiplyByDecimals(asset.decimals)
+  // @ts-ignore
```

Comment:
What are we ignoring?

Reply:
Same - `.decimalPlaces()` could result in `null`.